### ISSUE TITLE
Card colors for Badge

### ISSUE THREAD

**Issue by fabricionaweb** [zeit-ui/react#326]
# Feature request 🚀

  - [ ] I will create Pull Request
  - [x] It's just a suggestion

### Expected

We have in Cards more available colors (types) than in Badge.   
I would like to use the same color (like cyan) for Badge as well :D 

### Examples

```jsx
<Badge type="cyan"> ... </Badge>
```

**Comment by unix** [comment]
Yup, `Card` is special... 😿 
I'll  try to add some new colors to other components as well, but if you use different colors now, please follow this:

```jsx
import { useTheme } from '@zeit-ui/react'

() => {
 const theme = useTheme()

 return <Badge style={{ backgroundColor: theme.palette.cyan }}>1</Badge>
}
```

The `useTheme` contains all the built-in variables for the theme, you can use them freely in your own components. In addition, if there is any progress in my work, I'll update this issue again.

**Comment by fabricionaweb** [comment]
Thanks @unix I have done exactly this way :D

**Comment by unix** [comment]
I think custom colors are very simple, and there are many colors available, it would be cumbersome to add so many props to each component, so I added an example to show the user what to do.
Now we have four variations of each color(e.g. `successLight / successLighter / successDark` etc), it is not a good practice to enumerate these props on all components, so I want to leave these customized things to the users themselves. (let `type` prop be used only for TYPES, not colors)

And, `Card` is the exception, I didn't have a lot of colors when I created `Card`,  so this component is a little different. 😞 (now it looks like it's a bit out of line) I have tried and thought for some time that this is better, of course, any choice has its advantages and disadvantages. (while this is simple for most users, but it will let a few users do more)

I hope it helps you, and have a good day.
